include/libtorrent/aux_/win_util.hpp

```diff
@@ -38,16 +38,40 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace libtorrent { namespace aux {
 
+#ifdef TORRENT_WINRT
+	using LoadLibraryAPrototype = HMODULE WINAPI(_In_ LPCSTR lpLibFileName);
+#endif
+
 	template <typename Library>
 	HMODULE get_library_handle()
 	{
 		static bool handle_checked = false;
-		static HMODULE handle = nullptr;
+		static HMODULE handle;
```

Comment:
this doesn't look like an improvement. is there a reason not to initialize this?

Reply:
Thanks! I've unchanged it.

Reply:
Aren't statics zero-initialised? I suppose you would prefer something more explicit?

Reply:
global variables are, and it's possible this one will be too, as it's probably allocated in the BSS segment. But it's technically initialized at runtime, the first time this function is called.